python-meh.spec

```diff
@@ -17,133 +14,71 @@ Release: 1%{?dist}
 Source0: https://github.com/rhinstaller/python-meh/archive/%{name}-%{version}.tar.gz
 
 License: GPLv2+
-Group: System Environment/Libraries
 BuildArch: noarch
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
-BuildRequires: python-devel
 BuildRequires: gettext
-BuildRequires: python-setuptools
 BuildRequires: intltool
-BuildRequires: dbus-python
 BuildRequires: libreport-gtk >= %{libreportver}
 BuildRequires: libreport-cli >= %{libreportver}
-BuildRequires: libreport-python >= %{libreportver}
-BuildRequires: python-six
 
-%if 0%{with_python3}
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 BuildRequires: python3-dbus
-BuildRequires: libreport-python3 >= %{libreportver}
-BuildRequires: python3-six
-BuildRequires: python3-pocketlint
-%endif
-
-Requires: python
-Requires: dbus-python
-Requires: rpm-python
-Requires: libreport-cli >= %{libreportver}
-Requires: libreport-python >= %{libreportver}
-Requires: python-six
+BuildRequires: python3-libreport >= %{libreportver}
 
-%description
-The python-meh package is a python library for handling, saving, and reporting
+%global _description\
+The python-meh package is a python library for handling, saving, and reporting \
 exceptions.
 
-%package gui
-Summary: Graphical user interface for the python-meh library
-Requires: python-meh = %{version}-%{release}
-Requires: pygobject3
-Requires: gtk3
-Requires: libreport-gtk >= %{libreportver}
-
-%description gui
-The python-meh-gui package provides a GUI for the python-meh library.
+%description %_description
 
-%if 0%{with_python3}
 %package -n python3-meh
 Summary:  A python 3 library for handling exceptions
+%{?python_provide:%python_provide python3-meh}
+Obsoletes: python-meh <= 0.45
+Obsoletes: python2-meh <= 0.45
 Requires: python3
 Requires: python3-dbus
-Requires: rpm-python3
+Requires: python3-rpm
 Requires: libreport-cli >= %{libreportver}
-Requires: libreport-python3 >= %{libreportver}
-Requires: python3-six
+Requires: python3-libreport >= %{libreportver}
 
 %description -n python3-meh
 The python3-meh package is a python 3 library for handling, saving, and reporting
 exceptions.
 
 %package -n python3-meh-gui
 Summary: Graphical user interface for the python3-meh library
+%{?python_provide:%python_provide python3-meh-gui}
+Obsoletes: python-meh-gui <= 0.45
+Obsoletes: python2-meh-gui <= 0.45
 Requires: python3-meh = %{version}-%{release}
 Requires: python3-gobject, gtk3
 Requires: libreport-gtk >= %{libreportver}
 
-%description -n python3-meh-gui
+%description -n python3-%{srcname}-gui
```

Comment:
Looks like you have dropped the `srcname` everywhere else. You can drop it also.

Reply:
Dropped.